src/browserPlatform.ts

```diff
@@ -1,7 +1,9 @@
 import { StudentManager } from "./studentManager";
-import { getElementById } from "./utils";
+import { getElementById, ToastManager } from "./utils";
 export class BrowserPlatform {
-  constructor(public manager: any = new StudentManager()) {
+  #manager: StudentManager = new StudentManager();
+  private static toastManager = new ToastManager();
```

Comment:
it's not singleton. I would like use `ToastManger` like `ToastManger.instance.showSuccessMessage(...)` in any place your program